float64[] max_vel
---
int8 err
